generate-release-notes.py

```diff
@@ -1,20 +1,39 @@
 #!/usr/bin/env python3
-from github import Github  # https://github.com/PyGithub/PyGithub
-import yaml
+# pylint: disable=missing-module-docstring
```

Comment:
My preference would be to add docstrings to the relevant functions, even though they're mostly self-evident. Not a dealbreaker though.